traffic_stops/static/js/app/states/md/defaults.js

```diff
@@ -47,13 +47,11 @@ export default {
     'Safe Movement Violation': 1,
     'Vehicle Equipment Violation': 2,
     'Other Motor Vehicle Violation': 3,
-    'Investigation': 4,
```

Comment:
Since the crucial thing about this is the order, it doesn't matter that the sequence of numbers is now gappy, but it still grates on me; could you adjust the numbers so that they're a contiguous range?